packages/fxa-auth-server/test/remote/subscription_tests.js

```diff
@@ -154,14 +155,16 @@ describe('remote subscriptions:', function() {
       assert.deepEqual(result, []);
     });
 
+    // TODO: not sure if any changes need to be made here
```

Comment:
Yeah, looks like `display_name` is not returned, so there's nothing to assert here:

https://github.com/mozilla/subhub/blob/e530fcb67339c8b41035ec2a67b06d23c4ef97b2/subhub/subhub_api.yaml#L438-L479